lib/shelley/src/Cardano/Wallet/Shelley/Network/Blockfrost.hs

```diff
@@ -36,6 +36,7 @@ import Prelude
 import qualified Blockfrost.Client as BF
 import qualified Cardano.Api.Shelley as Node
 import qualified Data.Sequence as Seq
+import qualified Ouroboros.Consensus.HardFork.History.Qry as HF
 
 import Cardano.Api
     ( AnyCardanoEra, NetworkId (Mainnet, Testnet) )
```

Comment:
Sorry for not noticing earlier, I just noticed the type `NetworkId`. In the wallet, we already have a type with the same purpose, called `NetworkDiscriminant`.

I'm not entirely sure which one we want to keep, but I tend towards the `NetworkId` one for consistency with `Cardano.Api`. (I definitely want to get rid of the type-level uses of `NetworkDiscriminant`, but not necessarily the value level use).

For now, I think it's ok if we could try to avoid mixing the types too much.

Reply:
Ok, good to know!

The reason I chose `Cardano.NetworkId` not `NetworkDescriminant` is that its already used as a parameter in the `Node.withNetworkLayer` and is available in the local scope where `Blockfrost.withNetworkLayer` is also used. So it makes sense from the symmetry/consistency point of view.